local-proxy/src/server.ts

```diff
@@ -13,8 +13,9 @@ import os from 'os';
 import { initRegistry } from './stdio';
 import nanoid from 'nanoid';
 import mkdirp from 'mkdirp';
+import { copy } from 'fs-extra';
 
-const basePath = process.env.SHIFT_DEV_SERVER_BASE_REQUIRE_PATH;
+const basePath = process.env.SHIFT_DEV_SERVER_BASE_REQUIRE_PATH as string;
```

Comment:
Bit weird to cast as string but then do a truthy check on it. Why not cast after checking?

Reply:
Also there should be a generic way to do this.